### PR TITLE
Fix mixed content when using a load balancer

### PR DESCRIPTION
**Pull request by ADReece**
Fixes: https://github.com/DarkaOnLine/SwaggerLume/issues/126

### CODE REVIEW

File: config/swagger-lume.php [hunk 1]
```diff
@@ -201,4 +201,11 @@
     'constants' => [
         'SWAGGER_LUME_CONST_HOST' => env('SWAGGER_LUME_CONST_HOST', 'http://my-default-host.com'),
     ],
+    
+    /*
+    |--------------------------------------------------------------------------
+    | Force assets to be loaded over HTTPS (Solves mixed content errors when application is behind a load balancer.
+    |--------------------------------------------------------------------------
+     */
+    'force_https' => false,
```
Comment (DarkaOnLine) [inline config/swagger-lume.php:210]: Can we have `env` var for this:
```suggestion
    'force_https' => env('SWAGGER_LUME_FORCE_HTTPS', false),
```

File: src/helpers.php [hunk 2]
```diff
@@ -57,6 +57,6 @@ function swagger_lume_asset($asset)
             throw new SwaggerLumeException(sprintf('Requested L5 Swagger asset file (%s) does not exists', $asset));
         }
 
-        return route('swagger-lume.asset', ['asset' => $asset, 'v' => md5($file)], app('request')->secure());
+        return route('swagger-lume.asset', ['asset' => $asset, 'v' => md5($file)], !is_null(config('swagger-lume.force_https')) ? config('swagger-lume.force_https') : app('request')->secure());
```
Comment (DarkaOnLine) [inline src/helpers.php:60]:
```suggestion
        return route(
            'swagger-lume.asset',
            ['asset' => $asset, 'v' => md5($file)],
            config('swagger-lume.force_https') ?? app('request')->secure()
        );
```

**Review by DarkaOnLine — COMMENTED** [review]
Could we also add test and check if falg is sett that we serve assets with https?

File: config/swagger-lume.php (hunk 1 above)
Comment (ADReece) [reply config/swagger-lume.php]: Resolved in latest commit.

File: src/helpers.php (hunk 2 above)
Comment (ADReece) [reply src/helpers.php]: Resolved in latest commit.

**Comment by ADReece** [comment]
> Could we also add test and check if falg is sett that we serve assets with https?

Hi @DarkaOnLine I've added PhpUnit tests for the default, a true and a false value on the requested SWAGGER_LUME_FORCE_HTTPS flag.  Hopefully everything looks up to scratch now.

**Review by DarkaOnLine — APPROVED** [review]

**Pull request merged**